stable/admin/values.yaml

```diff
@@ -18,7 +18,7 @@ cloud:
   # cluster name is used to make resources unique in multi-cluster configurations.
   # If the NuoDB domain spans 2 or more physical clusters, then each cluster must have a unique clusterName.
   # The default is fine for single-cluster domains.
-  clusteName: cluster0
+  clusterName: cluster0
```

Comment:
So - previous changes indicate that you haven't merged this branch with master yet,

I am puzzled by this change.
Did we back away fro the new structure of having `cloud.cluster.name`?

Or is this change just a result of not having merged with master?

Or is this change an accidental result of a merge?

Reply:
The other branch is not yet merged with master, it's still on 4.0.4 will tidy it up when I do that one